app/models/case_contact.rb

```diff
@@ -16,6 +16,14 @@ class CaseContact < ApplicationRecord
     other
   ].freeze
   enum contact_type: CONTACT_TYPES.zip(CONTACT_TYPES).to_h
+
+  def humanized_type
+    "#{contact_type.humanize.titleize}#{ " - " + other_type_text if use_other_type_text?}"
```

Comment:
I love this! 😃

Reply:
I was thinking this would be a good candidate to refactor out into a view presenter object. It feels early to do that, though. I don't think there's enough complexity here to justify that yet